components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -653,6 +653,19 @@ def leave_none_unset_int(s):
           " ] or [\"Repository\", {\"viewname\": \"webindex\", "
           "\"query_string\": {\"experimenter\": -1}}, "
           "{\"title\": \"Repo\"}]'``")],
+    "omero.web.ui.metadata_pane":
+        ["METADATA_PANE",
```

Comment:
@will-moore @manics @pwalczysko @joshmoore could you have a look if you are ok with that?